notmuch/db.c

```diff
@@ -189,6 +184,11 @@ notmuch_database_t *nm_db_do_open(const char *filename, bool writable, bool verb
     }
   }
 
+  if (msg)
+  {
+    FREE(&msg);
```

Comment:
It's safe to call `FREE()` / `mutt_mem_free()` with a `NULL` pointer, so we don't need the `if (msg)`

Reply:
Neat. Fixed.